skimage/color/__init__.py

```diff
@@ -1,3 +1,5 @@
+"""Utilities for color conversion, color labeling, and color difference calculations."""
```

Comment:
```suggestion
"""Utilities for color conversion, color labeling, and color difference
calculations.
"""
```
Thanks @harshithakolipaka. Another small formatting tweak by me. We usually try to keep our line length within 79 characters according to [PEP8](https://peps.python.org/pep-0008/#maximum-line-length) (see also our [style guide](https://scikit-image.org/docs/dev/contribute.html#stylistic-guidelines)). I don't think our automation here checks this currently? But I thought it useful to point out. :)